bins/revme/Cargo.toml

```diff
@@ -16,17 +16,20 @@ indicatif = "0.17"
 plain_hasher = "0.2"
 revm = { path = "../../crates/revm", version = "3.5.0", default-features = false, features = [
     "ethersdb",
-    "std",
     "serde",
     "c-kzg",
 ] }
 alloy-rlp = { version = "0.3", default-features = false, features = [
     "arrayvec", 
     "derive",
 ] }
-serde = { version = "1.0", features = ["derive", "rc"] }
-serde_json = { version = "1.0", features = ["preserve_order"] }
+serde = { version = "1.0", default-features = false, features = ["derive", "rc"] }
+serde_json = { version = "1.0", default-features = false, features = ["alloc"]}
```

Comment:
`preserve_order` was a nice feature, does it work with `no_std`?

Reply:
it works, sry i added back at one place for miss the other :)

Reply:
I think `preserve_order` doesn't really work with `no-std` because of https://github.com/serde-rs/json/blob/master/Cargo.toml#L58 .  That's why I had to remove it in https://github.com/bluealloy/revm/commit/cbbfe9c9f2f3e17d4e67e979f954cd9e67b45753 in order to make `revme` no-std. But that should be fine in this repo because as pointed out it's only required by `revme` which is a separate tool, also I don't think it would be possible to fully make `revme` `no-std` because of `c-kzg`.